src/nodesessionmetadata.cpp

```diff
@@ -25,6 +25,17 @@
 
 namespace ovms {
 
+NodeSessionMetadata::NodeSessionMetadata() :
+    context({ExecutionContext::Interface::GRPC, ExecutionContext::Method::Predict}) {}
```

Comment:
Note that we need test for veryfying if rest requests in DAG will be properly noted in metrics due to this default constructor.